### ISSUE TITLE
Comments on columns running schema builder

### ISSUE THREAD

**Issue by rafael-renan-pacheco** [yajra/laravel-oci8#119]
Hi @yajra!

The following command does not add comment on the target column:

$table->string('my_column')->comment("my comment");

It is said [here](https://laracasts.com/discuss/channels/general-discussion/how-to-set-a-comment-on-table-column-using-schema-builder) and [here](http://stackoverflow.com/questions/28043799/laravel-schema-builder-setting-field-description) that this comment() method is allowed by the schema builder.

Can this method be added for Oracle as well?

Of course this can be achieved with pure SQL with DB::statement, but it would be much better with the comment() method.

Plus: Comments on table as well?

Best regards,
Rafael Pacheco.

**Comment by yajra** [comment]
@rafael-renan-pacheco, I am not aware that it's possible to add a comment now. Will have a look at this and will try to implement it. As far as I know, comment is not yet supported by the package.

**Comment by rafael-renan-pacheco** [comment]
Hi @yajra 

Since Oracle does not allow comments while creating a table, this modifier needs to be cached and executed after the table creation.

For example, `MySqlGrammar` returns a MySQL ready command to add comment on the desired column:
```
if (! is_null($column->comment)) {
    return ' comment "'.$column->comment.'"';
}
```

In `OracleGrammar`, this should be cached first in an array, and later executed after the table creation, to run `comment on column $table.$column is '$column->comment'`.

I don't know where the table is created to help any further :(

**Comment by yajra** [comment]
I am not using comment on my projects but will have a look at it. Can you submit PR if possible? You're on the right track of implementing it. Thanks!

**Comment by rafael-renan-pacheco** [comment]
Hi @yajra, done via #124 

This is my first PR ever, maybe I missed something because CI failed, but I don't think the error is related with my commit.

Also, can you please check if [this](https://github.com/yajra/laravel-oci8/pull/124/files#diff-53904ca4c0bbf9fab7f0d019370110fbR56) makes sense?

I did this to allow comment modification when changing some table, not only opon creation.

**Comment by yajra** [comment]
PR merged and released on v4.1.0. Thanks!
